### DISCUSSION TITLE
Error at upload after updating firmware

### DISCUSSION THREAD

**Discussion by Webfeger** [andycb/AdventurerClientJS#186] [Development]
There was a new firmware for my Adventurer 3 (1.3.1). After them i got an error message when i try to upload a file: File format error. Any ideas?

**Comment by andycb** [comment]
You're not the first to mention new firmware and issues. I hope Flashforge haven't made some move to block this from working, I guess the process is to start gathering information from everyone with issues. 

- Have you tried sending the same .g file to the printer using FlashPrint 4 (you can drag the already slices file in to send it without re-slicing) and seeing if the issue is with the file or the way it is sent?
- Are you able to grab some logs from the app, please? First is the app error logs stored here - Windows: `C:/Users/<userName>/AppData/Roaming/adventurer-client` or macOS: `~/Library/Logs/adventurer-client`
- The second logs are to enable debug mode in the app, send the file, then grab the trace of communication to / from the printer (this will include the sliced file, so I'd recommend slicing a simple cube to keep it short)

Thanks

**Comment by Webfeger** [comment]
Hello,
there was an update for flashprint (4.6.4) Sending a g-file also fails: 
![image](https://user-images.githubusercontent.com/37270376/134774214-9fe92d64-a3bc-4443-b855-1a04ab25e7c0.png)


But when i use Adventuerer Client, i see a progress bar on printer before i get an error message on printer.

I also tried to re-slice a file. but it issn't possible to send files by Flashprint anymore. Actually there are no successful ways to send a g-file over network.
The same g-files are working fine over usb-disk

[2021-09-25_log.log](https://github.com/andycb/AdventurerClientJS/files/7229862/2021-09-25_log.log)
[debug.txt](https://github.com/andycb/AdventurerClientJS/files/7229865/debug.txt)

**Comment by Webfeger** [comment]
Update:
I figured out that there was a bug in update function of  4.6.4 . The newest release is 5.1.1 and this version is working with the printer. It's possible to transfer the files over wifi.
But i can't use my formerly created g-files. It looks buggy and i can't send it to the printer. I have to re-slice it.

**Comment by andycb** [comment]
Thanks for sending over those logs and digging into this. The transfer log seems to imply that the printer accepted the file transfer and saved the file okay. So it looks like the printer is expecting something new in the G-Code. Do you have the same file sliced before/after the update?

**Comment by Webfeger** [comment]
Yea, i did use the same files. before and after the update. 
I don‘t know whats wrong with Flashprint because my g-files from Simplify 3D was working great in 4.6.4 or formerly versions. Now i cant use the g-files in FP and have to re-slice it. But when you transfer the same old g-file over usb, everything is working fine. So i guess we just have to check out the wifi transfer

**Comment by mvdkaap** [comment]
Same issue here, after upgrading firmware of FF Adventurer 3 to version 1.3.2, the Adventurer Client does send the .G-code files to the printer, but the printer displays 'File Format Error'.

**Reply by Gaweringo** [reply]
This happens to me if the actual G-Code in the file doesn't start before line 250 or something around that. This happens if there is a thumbnail embedded in the file.

**Comment by h45e** [comment]
This also is a problem with Adventurer 4

**Comment by yodaluca23** [comment]
This is happening to me with every software I try, Adventure client pops with the error before the printer even gets it, with FlashPrint it shows up I go to print and get same error, then it disappears??
